tox.ini

```diff
@@ -21,6 +21,15 @@ covercmd =
       --cover-branches \
       --nocapture
 
+[testenv:py27]
+basepython =
+    python2.7
+deps =
+    {[testenv]deps}
+    grpcio >= 0.13.0
```

Comment:
@jonparrott Does your `tox`-fu include knowing how to just say "use the `[grpc]` extra"?